swagger/swagger-generator/generator-core/src/main/java/org/apache/servicecomb/swagger/generator/pojo/PojoSwaggerGeneratorContext.java

```diff
@@ -47,6 +53,16 @@ public int getOrder() {
 
   @Override
   public boolean canProcess(Class<?> cls) {
+    for (Annotation annotation : cls.getAnnotations()) {
+      // we check the annotations by class name to avoid importing extra dependencies in this module
+      if ("javax.ws.rs.Path".equals(annotation.annotationType().getCanonicalName())
```

Comment:
RestController maybe scaned for some scenario. But I think it is fine for this implementation.